backend/common/src/main/java/ai/verta/modeldb/common/futures/FutureRest.java

```diff
@@ -14,23 +15,22 @@ public class FutureRest {
   private FutureRest() {}
 
   // Injects the result of the future into the grpc StreamObserver as the return of the server
-  public static <T> ResponseBodyEmitter ServerResponse(InternalFuture<T> f, Executor ex) {
-    ResponseBodyEmitter observer = new ResponseBodyEmitter();
+  public static <T> CompletableFuture<T> ServerResponse(InternalFuture<T> f, Executor ex) {
```

Comment:
method names should be lowercased. We're not barbarians writing go here!  ;)

Reply:
renamed it.